src/styles/blocks/main/_catalog.scss

```diff
@@ -0,0 +1,33 @@
+.catalog {
+  box-sizing: content-box;
+  display: grid;
+  max-width: 944px;
```

Comment:
seems like "magic" number, it's better to define it as a sum of variables (card-width * 4 + gap * 4, etc)